### PR TITLE
Cart_Page --designed

### PR DESCRIPTION
**Pull request by itsmebipul00**

### CODE REVIEW

**Review by DivyanshSareen — APPROVED** [review]
Good to merge

File: checkout.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+.cart-total {
+  flex-direction: column;
+  text-align: center;
+}
+
+.cart-total > * {
+  border: 5px solid hsl(var(--clr-dark));
+  padding: 0.5rem;
```
Comment (zoyatanweer) [inline checkout.css:36]: shouldn't it be in pixels?

File: checkout.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+@media (min-width: 60em) {
+  .cart-product-details {
+    background: hsl(var(--clr-white));
+    padding: 1rem;
+    margin: 1rem;
+    width: 70%;
+    display: grid;
+    gap: 0.5rem;
```
Comment (zoyatanweer) [inline checkout.css:50]: here too

**Review by zoyatanweer — APPROVED** [review]
looks good to merge with suggested changes.

**Review by Omkar-Ghate — APPROVED** [review]
Looks good for me.

**Pull request merged**